diff --git a/node_modules/jest-specific-snapshot/dist/index.js b/node_modules/jest-specific-snapshot/dist/index.js
index 2e59a1d..2056b2a 100644
--- a/node_modules/jest-specific-snapshot/dist/index.js
+++ b/node_modules/jest-specific-snapshot/dist/index.js
@@ -60,7 +60,8 @@ function toMatchSpecificSnapshot(received, snapshotFile) {
   if (!snapshotState) {
     snapshotState = new _jestSnapshot.SnapshotState(absoluteSnapshotFile, {
       updateSnapshot: commonSnapshotState._updateSnapshot,
-      snapshotPath: absoluteSnapshotFile
+      snapshotPath: absoluteSnapshotFile,
+      snapshotFormat: commonSnapshotState.snapshotFormat
     });
     snapshotsStateMap.set(absoluteSnapshotFile, snapshotState);
   }
